### PR TITLE
Fr/create workflow comment

### PR DESCRIPTION
**Pull request by sheehantoufiq**
<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @friggframework/api-module-ironclad@0.0.8-canary.63.3511cfb.0
  # or 
  yarn add @friggframework/api-module-ironclad@0.0.8-canary.63.3511cfb.0
  ```
</details>

### CODE REVIEW

File: api-module-library/ironclad/test/api.test.js [hunk 1]
```diff
@@ -128,13 +140,26 @@ describe('Ironclad API class', () => {
...
+            const body = {
+                creator: {
+                    type: 'email',
+                    email: 'projectteam@lefthook.com',
+                },
+                comment: 'Testing a comment',
+            };
+
+            const response = await api.createWorkflowComment(workflowID, body);
+            response.should.have.status(204);
```
Comment (JonathanEdMoore) [inline api-module-library/ironclad/test/api.test.js:161]: ` Invalid Chai property: status`

**Review by JonathanEdMoore — CHANGES_REQUESTED** [review]

File: api-module-library/ironclad/test/api.test.js (hunk 1 above)
Comment (sheehantoufiq) [reply api-module-library/ironclad/test/api.test.js:161]: @JonathanEdMoore Hmm, this works on my machine for some reason

Comment (sheehantoufiq) [reply api-module-library/ironclad/test/api.test.js:161]: Must be another global test library

**Review by JonathanEdMoore — CHANGES_REQUESTED** [review]
Getting an error in your `retrieve a record` test:
`{"code":"NOT_FOUND","message":"couldn't find a record matching the id workflow:63594312fbbf6c221e02bf1a"}`

**Review by JonathanEdMoore — APPROVED** [review]

**Comment by seanspeaks** [comment]
:rocket: PR was released in [`@friggframework/api-module-ironclad@0.0.9`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-ironclad%400.0.9) :rocket:

**Pull request merged**